### PR TITLE
Complete `novu/node` integrations module

### PR DESCRIPTION
**Pull request by codingmickey**
### What change does this PR introduce?

Completes the `novu/node` integrations module

### Why was this change needed?

For the completion of the `novu/node` package SDK #1319 

### Other information (Screenshots)

image

### CODE REVIEW

**Review by scopsy — APPROVED** [review]
Looks great @codingmickey!

File: packages/node/src/lib/integrations/integrations.spec.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    expect(mockedAxios.get).toHaveBeenCalled();
+    expect(mockedAxios.get).toHaveBeenCalledWith('/integrations');
+  });
+
+  test('should create an integration with the given parameters', async () => {
+    mockedAxios.post.mockResolvedValue({});
+
+    await novu.integrations.create('emailjs', {
+      credentials: {
+        apiKey: 'apikey',
```
Comment (scopsy) [inline packages/node/src/lib/integrations/integrations.spec.ts:34]:
```suggestion
        apiKey: 'api key',
```

File: packages/node/src/lib/integrations/integrations.spec.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        secretKey: 'apisecret',
```
Comment (scopsy) [inline packages/node/src/lib/integrations/integrations.spec.ts:35]:
```suggestion
        secretKey: 'api secret',
```

File: packages/node/src/lib/integrations/integrations.spec.ts [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+      active: true,
+      channel: ChannelTypeEnum.EMAIL,
+      check: true,
+    });
+
+    expect(mockedAxios.post).toHaveBeenCalled();
+    expect(mockedAxios.post).toHaveBeenCalledWith('/integrations', {
+      providerId: 'emailjs',
+      credentials: {
+        apiKey: 'apikey',
```
Comment (scopsy) [inline packages/node/src/lib/integrations/integrations.spec.ts:46]:
```suggestion
        apiKey: 'api key',
```

File: packages/node/src/lib/integrations/integrations.spec.ts [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        secretKey: 'apisecret',
```
Comment (scopsy) [inline packages/node/src/lib/integrations/integrations.spec.ts:47]:
```suggestion
        secretKey: 'api secret',
```

**Review by scopsy — COMMENTED** [review]
@codingmickey could you take a look at the spelling action?

**Comment by codingmickey** [comment]
Done!! 👍🏽

**Review by scopsy — APPROVED** [review]

**Pull request merged**